crates/forge_loader/src/manifest.rs

```diff
@@ -199,57 +321,278 @@ impl<T> AsRef<T> for FunctionTy<T> {
     }
 }
 
+
 impl<'a> ForgeModules<'a> {
-    pub fn into_analyzable_functions(
-        mut self,
-    ) -> impl Iterator<Item = FunctionTy<FunctionMod<'a>>> {
+
+// TODO: function returns iterator where each item is some specified type. 
+    pub fn into_analyzable_functions (
+        self,
+    ) -> Vec<Entrypoints<'a>>{
         // number of webtriggers are usually low, so it's better to just sort them and reuse
-        // the vec's storage instead of using a HashSet
-        self.webtriggers
-            .sort_unstable_by_key(|trigger| trigger.function);
-
-        // same rational for using a BTreeSet
-        let mut ignored_functions: BTreeSet<_> = self
-            .scheduled_triggers
-            .into_iter()
-            .map(|trigger| trigger.raw.function)
-            .chain(
-                self.event_triggers
-                    .into_iter()
-                    .map(|trigger| trigger.raw.function),
+        // self.webtriggers
+        //     .sort_unstable_by_key(|trigger| trigger.function);
+
+        // Get all the Triggers and represent them as a new struct thing where "webtrigger" attribute is true 
```

Comment:
This will create a lot of duplicate functions for the downstream analyzers to consume. It might be better to have a `BTreeSet` or a sorted `Vec` containing function names that are on user invokable modules. After you create it, iterate through all the functions appearing on the `functions` property and toggle the invokable flag if it appears in the set of invokable functions. Afterwards, toggle the web_trigger property if it appears as a webtrigger.